components/layouts/Layout.tsx

```diff
@@ -61,8 +61,7 @@ const Layout: React.FunctionComponent<Props> = props => {
                 <meta name="viewport" content="width=device-width, initial-scale=1.0" />
                 <link rel="preconnect" href="https://fonts.gstatic.com" />
                 <link
-                    rel="stylesheet preload prefetch"
-                    as="style"
+                    rel="stylesheet"
```

Comment:
This will hopefully address a console warning I was seeing in prod (but not dev while working on CSS refactor): `<link rel=preload> has an invalid `href` value`. This should fix that.